tests/src/test-save-geomodel.cpp

```diff
@@ -107,8 +107,10 @@ bool compare_double( std::string word1, std::string word2 )
     }
     try
     {
-        auto value1 = std::stol( word1 );
-        auto value2 = std::stol( word2 );
+        DEBUG( word1 );
```

Comment:
remove the two debugs

Reply:
Not sure yet, I am waiting for appveyor. This is the code I asked you to look at yesterday.

Reply:
I will remove them.